uid-pages/layout-bonita/src/assets/css/layout.css

```diff
@@ -11,42 +11,39 @@
      
     This CSS file manages the layout behaviour,   
     It allows to display the content container in full height, 
-    and header and footer with fixed size.
+    and header and footer fixed on the top and on the bottom of the page.
 
+    Note: All the CSS rules defined in this file override the values defined in UIDesigner properties panel.
 */
-
-html, body {
-    height: 100%;
-}
-
+html,
 body {
-    overflow: hidden;
+	height: 100vh;
 }
 
-body > div.container-fluid {
-    display: table;
-
-    width: 100%;
-    height: 100%;
-
-    margin: 0;
-    padding: 0;
-}
-
-body > div.container-fluid > div.row {
-    display: table-row;
+/*
+    Begin (flexbox vertical layout)
+    Force the layout-content to take all the available space between the layout-header and the layout-footer 
+*/
+body > .container-fluid {
+    height: inherit;
+    display: flex;
+    flex-direction: column;
 }
 
-body > div.container-fluid > div.row:nth-child(2),
+body > .container-fluid > div.row:nth-child(2),
 body > div.container-fluid > div.row:nth-child(2) div,
-body > div.container-fluid > div.row:nth-child(2) iframe {
-    height: 100%;
+body > div.container-fluid > div.row:nth-child(2) div > custom-living-application-i-frame-v4 {
```

Comment:
Is it not better to use an id to select Iframe ? Because if customer want use Bonita Layout... they can replace custom-widget..

Reply:
good catch, i pushed a fix for this